experimental/dds/attributable-map/src/map.ts

```diff
@@ -287,12 +287,21 @@ export class AttributableMap extends SharedObject<ISharedMapEvents> implements I
 
 		const data = this.kernel.getSerializedStorage(serializer);
 
-		// If single property exceeds this size, it goes into its own blob
-		const MinValueSizeSeparateSnapshotBlob = 8 * 1024;
+		// If single property exceeds this size, it goes into its own blob.
+		// Similar to below, there are no strict requirements for this value, but it should be reasonable.
+		// And similar, it does not impact much efficiency, other than small blobs add overhead.
+		const MinValueSizeSeparateSnapshotBlob = 128 * 1024;
 
 		// Maximum blob size for multiple map properties
 		// Should be bigger than MinValueSizeSeparateSnapshotBlob
-		const MaxSnapshotBlobSize = 16 * 1024;
+		// There is no strict requirement for this value, but it should be reasonable.
+		// Reasonably large, such that relative overhead of creating multiple blobs is not too high.
+		// Reasonably small, such that we don't create so large blobs that storage system has to split them.
+		// For example, ODSP stores content in 1Mb Azure blobs. That said, it stores compressed content, so the size of
+		// blobs has only indirect impact on storage size.
+		// Please note that smaller sizes increase the chances of blob reuse across summaries. That said
+		// we have no code on client side to do such dedupping. Service side blob dedupping does not help much (we still transfer bites over wire).
+		const MaxSnapshotBlobSize = 256 * 1024;
```

Comment:
Doing a quick mental check here, both old and new clients shouldn't need to worry about back compat here right? I'm assuming they both should be able to open these blobs no problem.

Reply:
Correct, they reading part does not care how big are the blobs, only writing part is affected